templates/app/node14-nestjs.yml

```diff
@@ -1,6 +1,6 @@
 services:
   app:
-    image: kooldev/node:14-nest
+    image: kooldev/node:16-nest
```

Comment:
we should rename this file to `node-nestjs.yml` and change its references as well